src/style.css

```
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.searchBar-form {
+  margin-top: 20px;
+  width: 100%;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.icon {
+  position: relative;
+}
+
+.icon::before {
+  content: "";
+  background: url(./images/Search.svg) no-repeat;
+  position: absolute;
+  top: 0;
+  height: 19px;
+  width: 19px;
+  left: 25px;
+}
+
+.input {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  font-family: Avenir, Arial, sans-serif;
```

```suggestion
  font-family: inherit;
```
використовуй наслідування